internal/wrappers/feature-flags.go

```diff
@@ -55,26 +58,70 @@ var FeatureFlagsBaseMap = []CommandFlags{
 }
 
 var FeatureFlags = map[string]bool{}
+var FeatureFlagsSpecific = map[string]bool{}
```

Comment:
I think it'll be better UsedFeatureFlags instead of FeatureFlagsSpecific

Reply:
Basically, the purpose of this map is a cache, so maybe FeatureFlagsCache?